### PR TITLE
Update all alternative pages

### PR DESCRIPTION
**Pull request by Davidslv**
Story: https://mojdigital.teamwork.com/#/tasks/20196427

### Screenshots

Screenshot 2020-04-16 at 16 15 09
Screenshot 2020-04-16 at 16 15 25
Screenshot 2020-04-16 at 16 14 44
Screenshot 2020-04-16 at 16 14 52
Screenshot 2020-04-16 at 16 14 58
Screenshot 2020-04-16 at 16 15 05

### CODE REVIEW

File: app/views/steps/alternatives/court/edit.en.html.erb [hunk 1]
```diff
@@ -1,74 +1,104 @@
...
+      Find out more about
+      <a href="https://www.refuge.org.uk/get-help-now/help-for-women/recognising-abuse/" rel="external" target="_blank">
+        domestic violence or abuse</a>
+      and
+      <a href="https://www.nspcc.org.uk/preventing-abuse/child-abuse-and-neglect/">child abuse</a>.
+    </p>
+
+
+    <%= step_form @form_object, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+      <%= f.govuk_radio_buttons_fieldset :court_acknowledgement, legend: { tag: 'span', size: 'm' } do %>
```
Comment (zheileman) [inline app/views/steps/alternatives/court/edit.en.html.erb:75]: We can do here the same as in `app/views/steps/miam/acknowledgement/edit.en.html.erb` and extract to locales the `label` and also the `fieldset legend`. I know we didn't have before a legend, but that was an accessibility failure. For the legend copy we can go, for example, with same as in the header: `Going to court`.

File: app/views/steps/alternatives/start/show.en.html.erb [hunk 2]
```diff
@@ -1,28 +1,27 @@
...
-      </ul>
-    </div>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>make the situation less stressful for the children</li>
+      <li>help the children continue family relationships</li>
+      <li>save time and money</li>
+      <li>reduce conflict with the other person</li>
+    </ul>
 
     <div class="xform-group form-submit">
```
Comment (zheileman) [inline app/views/steps/alternatives/start/show.en.html.erb:26]: Here we can now use the helper method to create this button-looking links. Take a look at `app/views/steps/safety_questions/start/show.en.html.erb` for an example. Also we can remove the wrapping div.

File: app/views/steps/alternatives/lawyer_negotiation/edit.en.html.erb [hunk 3]
```diff
@@ -1,58 +1,49 @@
...
+
+      <h3 class="govuk-heading-m">Cons</h3>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>can be a more expensive process</li>
+        <li>you may feel you’re not in control</li>
+        <li>can be seen as a confrontational approach</li>
+      </ul>
+    </div>
+
+    <p class="govuk-body">
```
Comment (zheileman) [inline app/views/steps/alternatives/lawyer_negotiation/edit.en.html.erb:43]: I think this link is part of the main content (left side) right before the radios. It also happens in some of the other pages.

File: app/views/steps/alternatives/court/edit.en.html.erb [hunk 4]
```diff
@@ -1,74 +1,103 @@
...
+      <a href="https://www.refuge.org.uk/get-help-now/help-for-women/recognising-abuse/" class="govuk-link" rel="external" target="_blank">
+        domestic violence or abuse</a>
+      and
+      <a href="https://www.nspcc.org.uk/preventing-abuse/child-abuse-and-neglect/" class="govuk-link" rel="external" target="_blank">child abuse</a>.
+    </p>
+
+
+    <%= step_form @form_object, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+      <%=
+        f.govuk_radio_buttons_fieldset :court_acknowledgement, legend: { tag: 'span', size: 'm' } do
```
Comment (zheileman) [inline app/views/steps/alternatives/court/edit.en.html.erb:76]: A small thing, it needs to be `govuk_check_boxes_fieldset`

File: app/views/steps/alternatives/start/show.en.html.erb [hunk 5]
```diff
@@ -1,31 +1,28 @@
 <% title 'Alternative ways to reach an agreement' %>
+<% step_header %>
 
-<div class="grid-row">
-  <div class="column-two-thirds">
-    <%= step_header %>
-
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
     <div class="grid_content_header">
```
Comment (zheileman) [inline app/views/steps/alternatives/start/show.en.html.erb:6]: I think we can get rid of the `grid_content_header ` div wrapper.

File: app/views/steps/alternatives/negotiation_tools/edit.en.html.erb [hunk 6]
```diff
@@ -1,59 +1,56 @@
...
+    <span class="govuk-caption-l">Alternative ways to reach an agreement</span>
+    <h1 class="govuk-heading-xl">Negotiation tools and services</h1>
+
+    <p class="govuk-body">
+      If you still communicate with the other person, and there are no concerns about domestic or child abuse, the
+      cheapest and easiest way to make arrangements is to negotiate with each other.
+    </p>
+    <p class="govuk-body">
+      There are free tools and services such as
+      <a href="https://www.splittingup-putkidsfirst.org.uk/home" class="govuk-link" rel="external" target="_blank">parenting plans</a>
```
Comment (zheileman) [inline app/views/steps/alternatives/negotiation_tools/edit.en.html.erb:17]: Let's change this link as per slack thread, as it is dead. We can use same or similar copy as in CAIT.

Comment (Davidslv) [reply app/views/steps/alternatives/negotiation_tools/edit.en.html.erb]: Updated on 3b8855f0cc9d4bf0af7d55359d977440d16a9f5f

**Comment by zheileman** [comment]
🎉 Thanks. Good catch on the second broken link, didn't know there were 2 of them 👍

**Pull request merged**